src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts

```diff
@@ -25,44 +25,24 @@ export class QuestionnaireFormFillerComponent implements OnInit {
 
   constructor(
     private route: ActivatedRoute,
-    private questionaireFillerServer: QuestionnaireFillerService,
+    private questionnaireFillerServer: QuestionnaireFillerService,
     private data: FhirConfigService
   ) {}
 
   ngOnInit() {
     this.questionnaire$ = this.route.paramMap.pipe(
       map((params: ParamMap) => params.get('id')),
-      map((name) =>
-        name === 'sdc-extract'
-          ? QuestionnaireDemo.getQuestionnaireSdcExtract()
-          : name === 'referral-min'
-          ? QuestionnaireDemo.getQuestionnaireReferralMin()
-          : name === 'sdc-cap'
-          ? QuestionnaireDemo.getQuestionnaireSdcCap()
-          : name === 'sdc-loinc'
-          ? QuestionnaireDemo.getQuestionnaireSdcLoinc()
-          : name === 'sdc-render'
-          ? QuestionnaireDemo.getQuestionnaireSdcRender()
-          : name === 'height-weight'
-          ? QuestionnaireDemo.getQuestionnaireLhncbHeightWeight()
-          : name === 'string'
-          ? QuestionnaireDemo.getQuestionnaireString()
-          : name === 'support-link'
-          ? QuestionnaireDemo.getQuestionnaireSupportLink()
-          : name === 'radiology-order'
+      map((id) =>
+        id === 'radiology-order'
           ? QuestionnaireDemo.getQuestionnaireRadiologyOrder()
-          : name === '-1'
-          ? this.questionaireFillerServer.getQuestionniare()
+          : id === '-1'
+          ? this.questionnaireFillerServer.getQuestionniare()
           : undefined
       )
     );
     this.questionnaireResponseInitial$ = this.route.paramMap.pipe(
```

Comment:
Do we still need this?

Reply:
good question. right now we don't have a local questionnaire response, so we can't use this, but in the future we might add one again. what do you think?

Reply:
I do not think that this is the case as this was primarily to test if the pre-filling via QuestionnaireResponse would work. In the new implementation we will pre-fill with the QuestionnaireResponses from the server which is more useful (I do not see any use case where a locally saved response would be preferable).